src/cli.rs

```diff
@@ -69,6 +89,10 @@ pub struct Args {
     /// set the label margin-y <px>
     #[arg(long)]
     pub label_margin_y: Option<i32>,
+
+    /// commands
```

Comment:
Maybe change this doc comment to something like "the selected command"?

Reply:
Fixed.